cmd/kubeadm/app/apis/kubeadm/v1alpha3/conversion_test.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha3_test
+
+import (
+	"reflect"
+	"testing"
+
+	"k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
+	"k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/scheme"
+	"k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1alpha3"
+	testutil "k8s.io/kubernetes/cmd/kubeadm/test"
+)
+
+func TestJoinConfigurationConversion(t *testing.T) {
+	testcases := map[string]struct {
+		old         *v1alpha3.JoinConfiguration
+		expected    *kubeadm.JoinConfiguration
```

Comment:
I think expected doesn't makes sense because we are checking only if it errors when expected